### DISCUSSION TITLE
Additional fields on ProfileScreen

### DISCUSSION THREAD

**Discussion by miloops** [firebase/FirebaseUI-Flutter#160] [Q&A]
Hey!

How are you handling the need of additional fields in ProfileScreen ? For example, I have user "country" that would like to be able to be shown/edited there.

Or the only way would be to copy/paste it and extend it myself on my app?

Thanks

**Comment by heki-dm** [comment]
Hello

I agree.

I want to save the added items in Firebase Authentication or Firebase Firestore.

I would be happy if I could register any type.

In addition to String, I would like to register something like List<String>.

Tanks.
